schemas/IDLs/org.gel.models.report.avro/6.0.0/CommonInterpreted.avdl

```diff
@@ -88,53 +92,496 @@ protocol CommonInterpreted {
+
+    /**
+    The reference genome assembly
+    */
+    enum Assembly {GRCh38, GRCh37}
+
+    enum ClinicalSignificance {
+        benign,
+        likely_benign,
+        likely_pathogenic,
+        pathogenic,
+        uncertain_significance
+    }
+
+    enum DrugResponseClassification {
+        altered_sensitivity,
+        reduced_sensitivity,
+        increased_sensitivity,
+        altered_resistant,
+        increased_resistance,
+        reduce_resistance,
+        increased_toxicity,
+        reduced_toxicity,
+        adverse_drug_reaction,
+        indication,
+        contraindication,
+        dosing_alteration,
+        increased_dosing,
+        reduced_dosing,
+        increased_monitoring,
+        increased_efficacy,
+        reduced_efficacy
+    }
 
+    enum PrognosisClassification{
+        altered_prognosis,
+        favorable_prognosis,
+        unfavorable_prognosis
+    }
+
+    enum TraitAssociation {
+        established_risk_allele,
+        likely_risk_allele,
+        uncertain_risk_allele,
+        protective
+    }
+
+    enum TumorigenesisClassification {
+        driver,
+        passenger,
+        modifier
+    }
+
+    enum VariantFunctionalEffect {
+        dominant_negative_variant,
+        gain_of_function_variant,
+        lethal_variant,
+        loss_of_function_variant,
+        loss_of_heterozygosity,
+        null_variant
+    }
+
+    enum Domain {
+        DOMAIN1, DOMAIN2, DOMAIN3, DOMAIN4, NONE
+    }
+
+    /**
+    The role of a given genomic feature in cancer
+
+* `NCIT_C16936`: oncogene. A gene that is a mutated (changed) form of a gene involved in normal cell growth. Oncogenes may cause the growth of cancer cells. Mutations in genes that become oncogenes can be inherited or caused by being exposed to substances in the environment that cause cancer. http://purl.obolibrary.org/obo/NCIT_C16936
+* `NCIT_C17362`: tumor_suppressor_gene. A type of gene that makes a protein called a tumor suppressor protein that helps control cell growth. Mutations (changes in DNA) in antioncogenes may lead to cancer. http://purl.obolibrary.org/obo/NCIT_C17362
+    */
+    enum RoleInCancer {
+        oncogene,
+        tumor_suppressor_gene,
+        both
+    }
+
+
+    /**
+    Each ACMG cirterion will be classifed as bening or pathogenic
+    */
+    enum AcmgEvidenceType{bening, pathogenic}
+    /**
+    Each ACMG criterion is weighted using the following terms:
+
+* `stand_alone`: `A`, stand-alone applied for benign variant critieria `(BA1)`
+* `supporting`: `P`, supporting applied for benign variant critieria `(BP1-6)` and pathogenic variant criteria `(PP1-5)`
+* `moderate`: `M`, moderate applied for pathogenic variant critieria (PM1-6)
+* `strong`: `S`, strong applied for pathogenic variant critieria (PS1-4)
+* `very_strong`: `S`, Very Stong applied for pathogenic variant critieria (PVS1)
+    */
+    enum AcmgEvidenceStrength{stand_alone, supporting, moderate, strong, very_strong}
```

Comment:
I would change this name to AcmgEvidenceWeight
to make it align with nomenclature in ACMG guidelines paper
https://www.nature.com/articles/gim201392 (Page 7, First column, last paragraph)

Would -in my opinion- ease the understanding of some other fields as commented below